core/src/main/java/com/novoda/noplayer/NoPlayerView.java

```diff
@@ -51,8 +47,8 @@ public View getContainerView() {
     }
 
     @Override
-    public SurfaceHolderRequester getSurfaceHolderRequester() {
-        return surfaceHolderProvider;
+    public PlayerSurfaceHolder getPlayerSurfaceHolder() {
+        return PlayerSurfaceHolder.create(surfaceView);
```

Comment:
Small nitpick but `getters` which do more than `get` should probably have a different name :smile_cat:   

Do we need to create a new `PlayerSurfaceHolder` each time the `get` is called or could we create it as part of `onFinishInflate`?

Reply:
Well spotted! It was the thing I _was planning to remember to move later_ 😄